add-on/chrome/modules/MozLoopAPI.jsm

```diff
@@ -188,8 +188,7 @@ const kMessageHandlers = {
     }
 
     let [windowId] = message.data;
-
-    win.LoopUI.startBrowserSharing();
+    win.LoopUI.startBrowserSharing(windowId);
```

Comment:
I would suggesting changing windowId to roomToken, or at least documenting that we're depending on that being true.